README.md

```diff
@@ -1,33 +1,64 @@
-# HVNC proposals and discussions
+# HGVS Nomenclature
 
-**Under construction**
+This repo is the home of the HGVS Nomenclature maintained by the Human Genome Organization Variation Nomenclature Committee.
 
-More instructions will follow.
+## Have Questions?
 
+Please use the [Discussions](discussions) forum to search for answers and ask new questions.
```

Comment:
Do we need a comment to state that some discussions may be closed? Are we in fact going to close discussions once they are acted on. I think just a couple of comments about navigation may help